### PR TITLE
Warning for wallet with incorrect network

### PR DESCRIPTION
**Pull request by KedziaPawel**
Needs translations

### CODE REVIEW

File: config/index.js [hunk 1]
```diff
@@ -9,11 +9,14 @@ try {
   isBeta = JSON.parse(IS_BETA);
 } catch (_) {}
 
+const networkName = BTCV_NETWORK || 'bitcoinvault';
```
Comment (PanRadaCloudBest) [inline config/index.js:12]: bitcoinvault const ?

File: class/legacy-wallet.js [hunk 2]
```diff
@@ -90,17 +90,13 @@ export class LegacyWallet extends AbstractWallet {
...
-    } catch (err) {
-      logger.error('legacy-wallet', `fetchBalance: ${err.message}`);
-    }
+    const balance = await BlueElectrum.getBalanceByAddress(this.getAddress());
+    this.balance = balance.confirmed + balance.unconfirmed;
+    this.unconfirmed_balance = balance.unconfirmed;
+    this.confirmed_balance = balance.balance;
+    this.incoming_balance = balance.alert_incoming;
+    this.outgoing_balance = balance.alert_outgoing;
+    this._lastBalanceFetch = +new Date();
```
Comment (PanRadaCloudBest) [inline class/legacy-wallet.js:99]: https://stackoverflow.com/questions/14255664/new-date-is-this-good-practice

Comment (KedziaPawel) [reply class/legacy-wallet.js]: done

File: config/index.js (hunk 1 above)
Comment (KedziaPawel) [reply config/index.js]: done

**Review by marek-siemieniuk-morawski — APPROVED** [review]

**Review by PanRadaCloudBest — APPROVED** [review]

**Pull request merged**